src/app/components/organization-info/index.js

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import Organization from './organization';
+import * as Constants from '../../constants/constants'
+import { withApollo } from "react-apollo";
+import { loader } from 'graphql.macro';
+import Header from './header';
+import UserInfo from '../user-info';
+
+const GET_ORG_INFO = loader('../../graphql/queries/organization-info.gql');
+const GET_USER_INFO = loader('../../graphql/queries/user-info.gql')
+
+const getRepositories = (client, searchName, query) => {
+  return client.query({
+    query: query,
+    variables: { searchName },
+  });
+};
+
+class HomePage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      path: '',
+      filterRepos: [],
+      search: null,
+      errors: null,
+      isOrganization: true,
+      selectedLanguages: [],
+    };
+  }
+
+  componentDidMount() {
+    const searchType = this.props.match.params.typeName;
+    const searchName = this.props.match.params.searchName;
+    if (searchName) {
+      if (searchType === 'Organization') {
+        this.fetchDataFromGitHub(searchName, GET_ORG_INFO);
+        this.setState({
+          path: searchName,
+          isOrganization: true,
+        })
+      } else if (searchType === 'User') {
+        this.fetchDataFromGitHub(searchName, GET_USER_INFO);
+        this.setState({
+          path: searchName,
+          isOrganization: false,
+        })
+      }
+    }
+  }
+
+  onChangeHandler = event => {
+    this.setState({ path: event.target.value });
+  };
+
+  onSubmitHandler = event => {
+    if (this.state.isOrganization) {
+      this.props.history.push(`/Organization/${this.state.path}`);
```

Comment:
@bhoomika08 always make sure that urls are in small case.

Reply:
Done.